pdk/1.1/modules/ROOT/pages/policies-pdk-upgrade-pdk.adoc

```diff
@@ -0,0 +1,107 @@
+= Upgrading PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway Policy Development Kit (PDK) is composed of the following separate versioned components that must each be updated to update a custom policy project:
```

Comment:
What about something like this? 

To update a custom policy project, you must update each of these separately versioned Flex Gateway Policy Development Kit (PDK) components: